src/pages/Cart/Cart.jsx

```diff
@@ -1,7 +1,43 @@
-import React from 'react'
-
+import React from 'react';
+import {useCart} from "../../context/cartcontext"
+import cartitems from '../../components/Cart/cartitems';
+import "./Cart.css"
+import { useAuth } from '../../context/auth-context';
 export const Cart = () => {
+  // const { cartItems } = useCart();
+const {auth} = useAuth();
+console.log(auth.user)
   return (
-    <div>Cart</div>
+    <div>
+      <div className=" card card_vertical"></div>
+      {/* {cartItems.map((item) => (
+          <CartItem key={item.id} cartitem={item} />
+        ))} */}
```

Comment:
Same Commented code

Reply:
I'll make sure that I don't repeat the same mistake